.github/workflows/tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    env:
+      FLOW_CONTEXT: Testing
+      PACKAGE_FOLDER: flow-base-distribution/DistributionPackages
+      FLOW_FOLDER: flow-base-distribution
+
+    runs-on: ubuntu-latest
+    name: PHP ${{ matrix.php-versions }} | Flow ${{ matrix.flow-versions }}
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.3', '7.4']
+        flow-versions: ['6.3']
+        dependencies: ['highest']
+
+    defaults:
+      run:
+        working-directory: ${{ env.FLOW_FOLDER }}
+
+    steps:
+    - name: Checkout Flow development distribution
+      uses: actions/checkout@v2
+      with:
+        repository: neos/flow-development-distribution
+        ref: ${{ matrix.flow-version }}
+        path: ${{ env.FLOW_FOLDER }}
+
+    - name: Extract package name from repository
+      run: echo "REPOSITORY_NAME=$(echo '${{ github.repository }}' | awk -F '/' '{print $2}')" >> $GITHUB_ENV
+      shell: bash
+
+    - name: Checkout package
+      uses: actions/checkout@v2
+      with:
+        path: ${{ env.PACKAGE_FOLDER}}/${$REPOSITORY_NAME}
+
+    - name: Setup PHP, with composer and extensions
+      uses: shivammathur/setup-php@v2 #https://github.com/shivammathur/setup-php
+      with:
+        php-version: ${{ matrix.php-versions }}
+        extensions: mbstring, xml, ctype, iconv, intl, pdo_sqlite, mysql
+        coverage: xdebug #optional
+
+    - name: Get composer cache directory
+      id: composer-cache
+      run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+    - name: Cache composer dependencies
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-php${{ matrix.php-versions }}-composer-${{ hashFiles('**/composer.json') }}
+        restore-keys: ${{ runner.os }}-php${{ matrix.php-versions }}-composer-
+
+    - name: Install dependencies
+      run: |
+        git -C ../${{ env.PACKAGE_FOLDER}}/${$REPOSITORY_NAME} checkout -b build 
```

Comment:
@albe We have a `$` to much here as well - (the GH actions says it can't substitute)